examples/ASDKgram/Sample/AppDelegate.m

```diff
@@ -0,0 +1,116 @@
+//
+//  AppDelegate.m
+//  ASDKgram
+//
+//  Created by Hannah Troisi on 2/16/16.
+//  Copyright © 2016 Hannah Troisi. All rights reserved.
+//
+
+#import "AppDelegate.h"
+#import "PhotoFeedViewController.h"
+#import "PhotoFeedNodeController.h"
+#import "Utilities.h"
+
+@interface AppDelegate () <UITabBarControllerDelegate>
+@end
+
+@implementation AppDelegate
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  
+  self.window                           = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
+  self.window.backgroundColor           = [UIColor whiteColor];
+  
+  // UIKit Home Feed viewController & navController
+  PhotoFeedNodeController *asdkHomeFeedVC     = [[PhotoFeedNodeController alloc] init];
+  UINavigationController *asdkHomeFeedNavCtrl = [[UINavigationController alloc] initWithRootViewController:asdkHomeFeedVC];
+  asdkHomeFeedNavCtrl.tabBarItem              = [[UITabBarItem alloc] initWithTitle:@"ASDK" image:[UIImage imageNamed:@"home"] tag:0];
+  asdkHomeFeedNavCtrl.hidesBarsOnSwipe        = YES;
+  
+  // ASDK Home Feed viewController & navController
+  PhotoFeedViewController *uikitHomeFeedVC     = [[PhotoFeedViewController alloc] init];
+  UINavigationController *uikitHomeFeedNavCtrl = [[UINavigationController alloc] initWithRootViewController:uikitHomeFeedVC];
+  uikitHomeFeedNavCtrl.tabBarItem              = [[UITabBarItem alloc] initWithTitle:@"UIKit" image:[UIImage imageNamed:@"home"] tag:0];
+  uikitHomeFeedNavCtrl.hidesBarsOnSwipe        = YES;
+
+  // UITabBarController
+  UITabBarController *tabBarController    = [[UITabBarController alloc] init];
+  tabBarController.viewControllers        = @[uikitHomeFeedNavCtrl, asdkHomeFeedNavCtrl];
+  tabBarController.selectedViewController = asdkHomeFeedNavCtrl;
+  tabBarController.delegate               = self;
+  
+  // Nav Bar appearance
+  [[UINavigationBar appearance] setBarTintColor:[UIColor darkBlueColor]];
+  [[UINavigationBar appearance] setTranslucent:NO];
+  NSDictionary *attributes = @{NSForegroundColorAttributeName:[UIColor whiteColor]};
+  [[UINavigationBar appearance] setTitleTextAttributes:attributes];
+  // make the status bar have white text (changes after scrolling, but not on initial app startup)
+  // UINavigationController does not forward on preferredStatusBarStyle calls to its child view controllers.
+  // Instead it manages its own state...http://stackoverflow.com/questions/19022210/preferredstatusbarstyle-isnt-called/19513714#19513714
+  uikitHomeFeedNavCtrl.navigationBar.barStyle = UIBarStyleBlack;
+  asdkHomeFeedNavCtrl.navigationBar.barStyle  = UIBarStyleBlack;
+  
+  self.window.rootViewController = tabBarController;
+  [self.window makeKeyAndVisible];
+  
+  // opaque status bar background
+  self.statusBarOpaqueUnderlayView                 = [[UIView alloc] init];
```

Comment:
You can get rid of the properties fro both window and the statusBar underlay and just use instance variables